packages/cardano-services-client/test/util.ts

```diff
@@ -62,46 +61,46 @@ export const getBobHandleProviderResponse = {
   profilePic: Asset.Uri('ipfs://zrljm7nskakjydxlr450ktsj08zuw6aktvgfkmmyw9semrkrezryq3yd1')
 };
 
-export const getAliceHandleAPIResponse: Partial<IHandle> = {
-  characters: 'rljm7n/23455',
-  created_slot_number: 33,
-  default_in_wallet: 'alice_default_hndle',
-  has_datum: false,
-  hex: '0f3abbc8fc19c2e61bab6059bf8a466e6e754833a08a62a6c56fe0e78f19d9d5',
-  holder: 'stake1uyehkck0lajq8gr28t9uxnuvgcqrc6070x3k9r8048z8y5gh6ffgw',
-  image: 'ipfs://c8fc19c2e61bab6059bf8a466e6e754833a08a62a6c56feasd',
-  length: 123,
-  name: 'alice',
-  numeric_modifiers: '-12.9',
-  og_number: 5,
-  rarity: Rarity.rare,
-  resolved_addresses: {
-    ada: 'addr_test1qqk4sr4f7vtqzd2w90d5nfu3n59jhhpawyphnek2y7er02nkrezryq3ydtmkg0e7e2jvzg443h0ffzfwd09wpcxy2fuqmcnecd'
-  },
-  standard_image: 'ipfs://c8fc19c2e61bab6059bf8a466e6e754833a08a62a6c56feasdfasd',
-  updated_slot_number: 22,
-  utxo: 'rljm7nskakjydxlr450ktsj08zuw6aktvgfkmmyw9semrkrezryq3ydtmkg0'
-};
+// export const getAliceHandleAPIResponse: Partial<IHandle> = {
```

Comment:
Why these two big commented blocks?

Reply:
I forgot to delete, addressed here: 895017eacb276da156d665a97af02ca16277fb13